src/components/Post/Post.tsx

```diff
@@ -1,15 +1,16 @@
 import React, {memo} from 'react';
-import {PostContainer, Title, Description, DateContainer} from "./post.styled";
+import {PostContainer, Title, Description, DateContainer, Button} from "./post.styled";
 type Props = IPost
 export const Post = memo(({id, title, description, createdAt}: Props) => {
   console.log("id: ", id);
   return (
       <PostContainer>
         <Title>{title}</Title>
```

Comment:
Hey!

I think you could add `ui` directory with components that do not use any business logic and have only stylings and presentations.

Could you please create `ui/Title` directory and add common `Title` component?